### ISSUE TITLE
Have "manage" with interval run first at "time 0"

### ISSUE THREAD

**Issue by philrz** [brimdata/zed#5028]
## tl;dr

When attempting to put the enhancement from #5017 to use to address https://github.com/brimdata/zui/issues/2847 I realized it would be helpful if `manage` performed its first run immediately.

## Details

Repro is with Zed commit 31ff542.

The effect is captured in the attached video below. Repro steps:

1. Start the Zed service via `zed -lake lake serve -manage 30s`
2. Quickly load many commits, such as by performing a separate `zed load` of each file in the [Zed sample data](https://github.com/brimdata/zed-sample-data/tree/main/zeek-default)
3. Perform a meta-query to confirm the presence of a separate data object for each load operation
4. Quit/restart the Zed service
5. Repeat the meta-query to confirm the same count of separate data objects
6. Wait the full 30 seconds for the first `manage` to trigger compaction
7. Repeat the meta-query to confirm the data has been compacted into fewer data objects

https://github.com/brimdata/zed/assets/5934157/9d0748a1-8c1b-424a-b5e9-baae1af7c311

Now let's think about the same effect as it might apply to https://github.com/brimdata/zui/issues/2847. To avoid excess chatter in the logs, I'd been planning to have Zui assign a conservative `manage` interval on the Zed service that it spawns, e.g., perhaps 1 hour. Since a user that launches Zui may jump right to performing queries, I figure it would be a nice improvement if that first `manage` compaction trigger happened at "time 0" as well, i.e., as soon as the `zed serve` process launches. Without this, a user that happens to only work in brief Zui sessions would never see compaction benefits from their data. For users that work in longer sessions, now they wouldn't have to wait an hour to get best case query perf. Also, a user that becomes hip to this could also exit/restart Zui to trigger that same "time 0" compaction if they should ever perform a big data load and want the compaction boost right away.

**Comment by philrz** [comment]
We discussed this one as a team. The main argument against this approach is that it could potentially introduce significant additional load at the time the Zed service is starting. Meanwhile, others didn't seem to have the concerns I had about log chattiness (which may be separately addressed regardless) and thought it would be ok to just lower the manage interval to something brief like 5 minutes. Therefore I'm game to close this issue and save us the extra work at the Zed layer and will instead have Zui invoke manage at a 5 minute interval for now.
